dlp/inspectWithHotwordRules.js

```diff
@@ -0,0 +1,174 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//  title: Inspects strings
+//  description: Inspect a string using the Data Loss Prevention API.
+//  usage: node inspectWithHotwordRules.js my-project string minLikelihood maxFindings infoTypes customInfoTypes includeQuote hotwordRegexPattern
+
+function main(
+  projectId,
+  string,
+  minLikelihood,
+  maxFindings,
+  infoTypes,
+  customInfoTypes,
+  includeQuote,
+  hotwordRegexPattern
+) {
+  [infoTypes, customInfoTypes] = transformCLI(infoTypes, customInfoTypes);
+  // [START dlp_inspect_hotword_rule]
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  // The project ID to run the API call under
+  // const projectId = 'my-project';
```

Comment:
why do you have these things commented out when you pass them in via variables?

Reply:
The parameters will not be shown to users reading the docs. The commented lines will help them know what things they need to define before running the sample.